### PR TITLE
Set up/structure

### PR DESCRIPTION
**Pull request by Jurgis586**
Nekreipk demesio i ta antra kur Merge branch, misspellinau 🗡

### CODE REVIEW

File: frontend/src/Components/testComponent/testComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class TestComponent extends Component {
```
Comment (LiudasD) [inline frontend/src/Components/testComponent/testComponent.jsx]: component name is okay, but why file name is small capital?

File: frontend/src/GlobalCSS.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:root{
+    --gembird-text-color: #000000;
```
Comment (LiudasD) [inline frontend/src/GlobalCSS.css]: why do you need to specify "gembird" everywhere?

Comment (Jurgis586) [reply frontend/src/GlobalCSS.css]: Because its gembird project, Liudas Demikis.

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
+import { TestComponent } from './Components';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <h1>project-gembird on init...</h1>
```
Comment (LiudasD) [inline frontend/src/App.js]: <Route path={siteRoutes.ABOUT} component={AboutPage} />

example of how to link route to component

**Pull request merged**